app.js

```diff
@@ -1,88 +1,813 @@
 // Task 1 - Notes for JS Evaluation Predictions
+//4 + 4 / 2 -> 6
+// (2 + -7) * 3 -> 15
+// 101 % 10 -> 1
+// 12 - 4 % 3 -> 11
+// true && false -> false
+// true && !(false || false) -> true
+// !true && !(false || false) - > false
+// 'cat' + 'dog' -> 'catdog'
+// 2 + 'pizza' -> 2pizza
+// 2.5 * 'fish' -> NaN
+// 5 >= 11 -> false
+// 5 === 5.0 -> true
+// 7 !== 7.1 -> true
+// 5 + 5 > 8 -> true
+// 6 + 6 !== 12 -> false
+// 2 > 1 || false
+// 'true' === true
+// 10 % 2 === 0 -> true
+// 21 % 2 === 0 -> false
+// 21 % 2 !== 0 -> false
+// 21 % 2 === 1 -> true
+// 12 % 3 === 0 -> true
+// 9 % 3 === 0 -> true
+// 14 % 3 === 0 -> false
+// 'new york'[0] -> 'n'
+// 'new york'[1] -> 'e'
+// 'san francisco'[2 * 3] -> 'a'
+// 'engineering'[3].toUpperCase() -> 'I'
+// 'engineering'.indexOf('G') -> -1
+// 'engineering'.indexOf('neer') -> 4
+// 'engineering'.indexOf('r') > -1 -> true
+// 'science'.indexOf('x') === -1 -> true
 
 // List 3 expressions that surprised you or gave unexpected results, why?
+// 2 + 'pizza' surpised me because I figured it would have been undefined, my thought process was you cant add a string a number together
+// 6 + 6 !== 12 this one surprised me because 6+6 = 12 im still kind of confused on it
+// 'engineering'.indexOf('G') -> -1 i figured this one would output [2,10]
 
 // List 2 exprsssions you felt comfortable interpreting at a glance.
+// (2 + -7) * 3 -> 15
+// 101 % 10 -> 1
 
 // List 1 expression you are still 'shaky' on, why?
-
+// science'.indexOf('x') === -1 -> true i dont undsertand how this equates to true
```

Comment:
This code is looking for the index of `x` in the string `science`. because `x` does not exist in that word, you get returned `-1`. `-1` usually means whatever you're looking for can't be found.